### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by sualko**

### CODE REVIEW

File: src/Core/Meeting.php [hunk 1]
```diff
@@ -379,6 +379,36 @@ public function getAttendees()
         return $this->attendees;
     }
 
+    /**
+     * Moderators of Meeting - Subset of Attendees
+     * @return Attendee[]
+     */
+    public function getModerators()
```
Comment (FelixJacobi) [inline src/Core/Meeting.php:386]:
```suggestion
    public function getModerators(): array
```

I guess we should add return type hints when pulling upstream changes and their miss them. As our library has an independent release schedule and you must decide to use it, I guess that is not a big problem.

File: src/Core/Meeting.php [hunk 2]
```diff
@@ -379,6 +379,36 @@ public function getAttendees()
...
+        });
+
+        return array_values($moderators);
+    }
+
+    /**
+     * Viewers of Meeting - Subset of Attendees
+     * @return Attendee[]
+     */
+    public function getViewers()
```
Comment (FelixJacobi) [inline src/Core/Meeting.php:401]:
```suggestion
    public function getViewers(): array
```

File: tests/unit/Responses/GetMeetingInfoResponseTest.php [hunk 3]
```diff
@@ -87,6 +88,36 @@ public function testMeetingAttendeeContent()
         $this->assertEquals('a:focus{color:#0181eb}', $customData['customStyle']);
     }
 
+    public function testMeetingModerators()
```
Comment (FelixJacobi) [inline tests/unit/Responses/GetMeetingInfoResponseTest.php:91]:
```suggestion
    public function testMeetingModerators(): void
```

File: tests/unit/Responses/GetMeetingInfoResponseTest.php [hunk 4]
```diff
@@ -87,6 +88,36 @@ public function testMeetingAttendeeContent()
...
+        $firstModerator = $moderators[0];
+        $this->assertEquals('Ernie Abernathy', $firstModerator->getFullName());
+        $this->assertEquals('MODERATOR', $firstModerator->getRole());
+
+        $secondModerator = $moderators[1];
+        $this->assertEquals('Barrett Kutch', $secondModerator->getFullName());
+        $this->assertEquals('MODERATOR', $secondModerator->getRole());
+    }
+
+    public function testMeetingViewers()
```
Comment (FelixJacobi) [inline tests/unit/Responses/GetMeetingInfoResponseTest.php:106]:
```suggestion
    public function testMeetingViewers(): void
```

**Review by FelixJacobi — CHANGES_REQUESTED** [review]
Looks fine for me except for the type hints.

**Comment by sualko** [comment]
I added all your suggested type hints in one commit.

**Comment by sualko** [comment]
@SamuelWei @FelixJacobi we can include this in 4.0, right?

**Comment by FelixJacobi** [comment]
> @SamuelWei @FelixJacobi we can include this in 4.0, right?

In think so :) .

**Review by FelixJacobi — APPROVED** [review]

**Pull request merged**